minecraft_servers/drazyh/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+  "server_name": "drazyh",
+  "nice_name": "Drazyh",
+  "direct_ip": "drazyh.com",
+  "gamemodes": {
+    "p4free": {
+      "name": "P4 Free",
+      "color": "#f89900"
+    },
+    "rankup": {
+      "name": "RankUP",
+      "color": "#f89900"
+    },
+    "factions": {
+      "name": "Factions",
+      "color": "#f89900"
+    }
+  },
+  "social": {
+    "discord": "https://discord.com/invite/gdmqvRgzGf",
+    "web_shop": "https://loja.drazyh.com"
```

Comment:
Not reachable for me

Reply:
It must be a DDoS protection issue from our provider (cubeShop), but I have removed the "web_shop". I believe that's what you were unable to access.